src/handlers/showcase.ts

```diff
@@ -12,8 +12,17 @@ export default async function handleShowcaseMessage(
     const showcaseChannel = (await client.channels.fetch(
       config.FORTIES_SHOWCASE
     )) as TextChannel;
+    let url;
 
-    const url = msg.attachments.first()?.url as string;
+    if (msg.attachments.size > 1) {
+      const acceptedFileFormats = new RegExp('.(jpe?g|png|gif|bmp)$', 'i');
+      const attachmentToUse = msg.attachments.find((attachment) =>
+        acceptedFileFormats.test(attachment.url)
+      );
+      if (attachmentToUse) url = attachmentToUse.url ;
+    } else {
+      url = msg.attachments.first()?.url as string;
+    }
```

Comment:
I think we can simplify this. The `if ... else ...` in this case is redundant. If we use `find()` it will get us the first item in the array that matches the condition, or none if that's the case, whether the array contains 1 element or 100. You've actually already done that:

```suggestion
      const acceptedFileFormats = new RegExp('.(jpe?g|png|gif|bmp)$', 'i');
      const url = msg.attachments.find((attachment) =>
        acceptedFileFormats.test(attachment.url)
      )?.url;
```

Reply:
> I think we can simplify this. The `if ... else ...` in this case is redundant. If we use `find()` it will get us the first item in the array that matches the condition, or none if that's the case, whether the array contains 1 element or 100. You've actually already done that:

Whoops 🤦🏼‍♂️ that's what I get for trying to knock this out in a few minutes and rushing it lol. I think in my head I was trying to avoid unnecessary loops if there was more than one attachment, but this array will never be large so that's a moot point anyways. I'll change that.